scripts/build-micropython.sh

```diff
@@ -75,6 +75,7 @@ make V=1 -C $(realpath ../../../../third_party/micropython/litex/) -j$JOBS
 cd $OLD_DIR
 
 # Generate a firmware image suitable for flashing.
-#python -m litex.soc.tools.mkmscimg -f $TARGET_MPY_BUILD_DIR/firmware.bin -o $TARGET_MPY_BUILD_DIR/firmware.fbi
-#/usr/bin/env python mkimage.py $MISOC_EXTRA_CMDLINE $LITEX_EXTRA_CMDLINE --output-file=$TARGET_BUILD_DIR/micropython.bin --override-firmware=$TARGET_MPY_BUILD_DIR/firmware.fbi
 make image
+#
+python -m litex.soc.tools.mkmscimg -f $TARGET_MPY_BUILD_DIR/firmware.bin -o $TARGET_MPY_BUILD_DIR/firmware.fbi
+/usr/bin/env python mkimage.py $MISOC_EXTRA_CMDLINE $LITEX_EXTRA_CMDLINE --output-file=$TARGET_BUILD_DIR/micropython.bin --override-firmware=$TARGET_MPY_BUILD_DIR/firmware.fbi
```

Comment:
I don't think these lines should be needed anymore? You should end up with a `$TARGET_BUILD_DIR/gateware+bios+micropython.bin` file from the `make image` command?

Reply:
Updated `scripts/build-micropython.sh` to check `FIRMWARE=micropython` is set (thanks for hint as to what `scripts/build-linux.sh` does), which does result in `image-gateware+bios+micropython.bin` being built by "`make image`" when it is set (and also results in `make image-flash` doing the right thing).  Removed the lines you commented on which are now unnecessary.   Updated [blog post](http://ewen.mcneill.gen.nz/blog/entry/2018-01-17-fupy-fpga-micropython-on-mimas-v2-and-arty-a7/) to include extra environment variable.